src/index.html

```diff
@@ -5,8 +5,406 @@
+        </div>
+        <div class="card__items card_code">
+          Код товара: 195434
+        </div>
+        <div class="card__items">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class="card_text">
+            Отзывов: 5
+          </span>
+        </div>
+        <div class="card__items card__items_price">
+          <span class="card_text">
+            Цена:
+          </span>
+          <span class="card_value">
+            69 999 грн
+          </span>
+        </div>
+        <div class="card__items">
+          <a
+            class="button"
+            href="#"
+          >
+            Купить
+          </a>
+        </div>
+      </div>
+
+      <div class="catalog__card card" >
+        <img
+          class="card_image"
+          src="./images/imac.jpeg"
+          alt="imac"
+        >
+        <div class="card__items card_title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </div>
+        <div class="card__items card_code">
+          Код товара: 195434
+        </div>
+        <div class="card__items">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class="card_text">
+            Отзывов: 5
+          </span>
+        </div>
+        <div class="card__items card__items_price">
+          <span class="card_text">
+            Цена:
+          </span>
+          <span class="card_value">
+            69 999 грн
+          </span>
+        </div>
+        <div class="card__items">
+          <a
+            class="button"
+            href="#"
+          >
+            Купить
+          </a>
+        </div>
+      </div>
+
+      <div class="catalog__card card" >
+        <img
+          class="card_image"
+          src="./images/imac.jpeg"
+          alt="imac"
+        >
+        <div class="card__items card_title">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </div>
+        <div class="card__items card_code">
+          Код товара: 195434
+        </div>
+        <div class="card__items">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class="card_text">
+            Отзывов: 5
+          </span>
+        </div>
+        <div class="card__items card__items_price">
+          <span class="card_text">
```

Comment:
Елементи розділяй двома дефісами
```suggestion
          <span class="card__text">
```